workspace/__lib__/xod/bits/i16-to-number/patch.cpp

```diff
@@ -0,0 +1,13 @@
+
+struct State {
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    uint8_t b1 = getValue<input_B1>(ctx);
+    uint8_t b0 = getValue<input_B0>(ctx);
+    int16_t bytes = ((b1 << 8) | b0);
+    Number result = bytes;
```

Comment:
The outer parens are not necessary. I suggest being more explicit on why the `Number result = bytes` statement exists. You perform casting here, so:

```cpp
Number result = (Number)bytes;
// or
auto result = (Number)bytes;
// or
auto result = static_cast<Number>(bytes);
```

The same for other implementations. Just be consistent.